### PR TITLE
Initial commit for Comet PHP framework

### PR DESCRIPTION
**Pull request by gotzmann**
Initial commit with ALL Techempower Benchmarks for Comet PHP framework: https://github.com/gotzmann/comet

### CODE REVIEW

**Comment by gotzmann** [comment]
I'm not sure why Travis fails on this string:

TESTDIR="PHP/php PHP/comet"

Maybe that's because folder "frameworks/PHP/comet" doesn't exist in master branch?

File: .travis.yml [hunk 1]
```diff
@@ -62,8 +62,9 @@ env:
     - "TESTLANG=Nim"
     - "TESTLANG=Perl"
     - 'TESTDIR="PHP/php"'
+    - 'TESTDIR="PHP/comet"'
```
Comment (joanhey) [inline .travis.yml:65]: Don't create a new job for only 1 framework.
Please, revert that change.

**Comment by joanhey** [comment]
It's the same where you place comet, always will show you that fail the job.
If you click in that job, you will see why it is failing.
https://travis-ci.org/github/TechEmpower/FrameworkBenchmarks/jobs/692627577#L1292

File: .travis.yml (hunk 1 above)
Comment (gotzmann) [reply .travis.yml]: Done

**Comment by gotzmann** [comment]
I've fixed Travis warnings. All checks have passed. Woohoo!

**Pull request merged**